New Entries: Pandas `GroupBy` and `.groupby()` (#774)
Codecademy/docs: content/pandas/concepts/groupby/terms/count/count.md

---
Title: '.count()'

> Description: 'Produces a result with counts of the values for each group in a GroupBy object.'

Reviewer: ```suggestion
Description: 'Produces a new Series or DataFrame with counts of the values for each group in a GroupBy object.'
```

Reviewer: We typically mention the data type in these docs, but for all these methods there is no discussion of the return type, only that a 'result' is returned. I don't think it has to be in the description but I think somewhere it should be stated that the return will be a series or dataframe.

Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Data Structures'
  - 'Pandas'
CatalogContent:
  - 'learn-python-3'
  - 'paths/data-science'
---

> The `.count()` method produces a new Series or [DataFrame](https://www.codecademy.com/resources/docs/pandas/dataframe) with counts of the values for each group in a in a [GroupBy](https://www.codecademy.com/resources/docs/pandas/groupby) object.

Reviewer 2: ```suggestion
The `.count()` method produces a new `Series` or [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) with counts of the values for each group in a in a [`GroupBy`](https://www.codecademy.com/resources/docs/pandas/groupby) object.
```

## Syntax

```pseudo
groupbyobject.count()
```

## Example

> The following example produces a GroupBy object from a [DataFrame](https://www.codecademy.com/resources/docs/pandas/dataframe) and executes the `.count()` method on it.

Reviewer 2: ```suggestion
The following example produces a `GroupBy` object from a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) and executes the `.count()` method on it.
```

---
Title: '.count()'
Description: 'Produces a result with counts of the values for each group in a GroupBy object.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Data Structures'
  - 'Pandas'
CatalogContent:
  - 'learn-python-3'
  - 'paths/data-science'
---

The `.count()` method produces a result with counts of the values for each group in a in a [GroupBy](https://www.codecademy.com/resources/docs/pandas/groupby) object.

## Syntax

```pseudo
groupbyobject.count()
```

## Example

The following example produces a GroupBy object from a [DataFrame](https://www.codecademy.com/resources/docs/pandas/dataframe) and executes the `.count()` method on it.

```py
import pandas as pd

df = pd.DataFrame({'Key' : ['A', 'A', 'A', 'B', 'B', 'C'],
                   'Value' : [15., 23., 17., 5., 8., 12.]})

> print(df)

Reviewer 3: ```suggestion
print(df, end='\n\n')
```

group = df.groupby(['Key'])

print(group.count())
```

This example produces the following output:

```shell

>   Key  Value
> 0   A   15.0
> 1   A   23.0
> 2   A   17.0
> 3   B    5.0
> 4   B    8.0
> 5   C   12.0
> 
>      Value
> Key
> A        3
> B        2
> C        1

Reviewer 3: ```suggestion
  Key  Value
0   A   15.0
1   A   23.0
2   A   17.0
3   B    5.0
4   B    8.0
5   C   12.0

  Key  Value
0   A      3
1   B      2
2   C      1
```
